apiary.apib

```diff
@@ -417,3 +417,66 @@ Returns the sum of runtime in seconds of all jobs in the project.
                     "durationSum": 12345
                 }
             }
+
+# Group Configuration
+
+Configuration utilities
+
+## Resolve Configuration [/configuration/resolve]
+
++ Attributes
+    + componentId (required, string) - Component ID
+    + configurationId (required, string) - Configuration ID
+    + configVersion (required, string) - Configuration Version
+    + variableValuesId (optional, string) - ID of values of variables referenced in the configuration (if any)
+    + variableValuesData (optional, array) - [Actual values](https://developers.keboola.com/integrate/variables/#option-3--run-a-job-with-inline-values) of variables references in the configuration
+
+### Resolve Configuration [POST]
+
+Processes a configuration as if it were executed. This involves replacing
+[variables](https://developers.keboola.com/integrate/variables/) and
+[shared code](https://developers.keboola.com/integrate/variables/#shared-code). For configurations with rows, this also
+includes [merging configuration and configuration rows](https://help.keboola.com/components/#configuration-rows).
+The processed configuration is returned. If the configuration has rows, then the configuration itself is
+unmodified (it would not be executed). The state on both configuration and configuration rows is always returned as
+empty. The provided configuration is not unmodified, modifications are only returned.
```

Comment:
Tato posledna veta mi nedava zmysel:
`The provided configuration is not unmodified, modifications are only returned.`

Nemalo by tam byt `not unmodified` -> `not modified` alebo len `unmodified`?

Reply:
Treba `The provided configuration has not been modified, only modifications are returned.`?

Reply:
nj, to byl nejakej zkrat

Reply:
dam tam `The provided configuration is not modified -- modifications are not stored, only returned` ?

Reply:
not sure I understand :-) .  If the provided configuration is not modified then there are no modifications to store, so the original unmodified configuration is returned?
Is that what it does?

Reply:
The modified configuration is returned in the response, but it is not stored.

Reply:
normally when you do a POST on configuration, the api changes the configuration and returns it in the response - i want to emphasize that here the api never changes the configuration

Reply:
I understand now, thank you.  I would put there then `The provided configuration modifications are not stored, but they are returned in the response.`

Reply:
yeah, that's good :)